content/pages/1.install.md

```diff
@@ -34,24 +38,27 @@ The easiest way install ``mycli`` on a Mac is to use [homebrew].
 
 [https://packages.debian.org/search?keywords=mycli]
 
-```
+    :::bash
     $ sudo apt-get update
     $ sudo apt-get install mycli
-```
 
-### RHEL, Centos, Fedora:
+### Fedora
+
+    :::bash
```

Comment:
Hmm, yeah, I think that's how Pelican does code blocks. Github-flavored Markdown is different.

To be honest, I was imitating the triple-colons that were already in the file. Looking at [Pelican's docs](http://docs.getpelican.com/en/3.6.3/content.html#syntax-highlighting), it seem like what they recommend (and when you run the development server, it renders).

<img width="248" alt="screen shot 2017-06-02 at 21 18 16" src="https://cloud.githubusercontent.com/assets/3047055/26749820/08347c2e-47d9-11e7-8c2b-a9f13e69f66e.png">